sdk-tests/pom.xml

```diff
@@ -145,6 +145,12 @@
       <version>${dapr.sdk.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
This is needed for the tests to work on IntelliJ. Not sure why it is like that in master branch.